config/settings/base.py

```diff
@@ -187,10 +187,13 @@
 DEFAULT_FROM_EMAIL = SERVER_EMAIL
 EMAIL_TIMEOUT = 5
 
-# For sending Front email. Only used for Payouts.
+# For sending email through Front.
 FRONT_TOKEN = env("FRONT_TOKEN", default=None)
 FRONT_CHANNEL = env("FRONT_CHANNEL", default=None)
 FRONT_AUTHOR = env("FRONT_AUTHOR", default=None)
+FRONT_SENDER_NAME = env("FRONT_SENDER_NAME", default=None)
```

Comment:
Should this default to `DEFAULT_FROM_EMAIL`?

Reply:
This actually isn't an email address. It is just a display name. The "from" address will always be configured by what the address is from `FRONT_CHANNEL`. For payouts, this is set to "EthicalAds by Read the Docs". I think that's a reasonable value.